src/main/java/com/tstepnik/planner/controller/RegistrationController.java

```diff
@@ -29,7 +29,7 @@ public RegistrationController(RegistrationService registrationService, RegisterU
 
     @PostMapping("/register")
     public ResponseEntity<UserRegisterDTO> registerUser(@RequestBody UserRegisterDTO user) {
-        Optional<User> registerUser = Optional.ofNullable(registrationService.register(mapper.toUser(user)));
+        Optional<User> registerUser = Optional.ofNullable(registrationService.register(mapper.toEntity(user)));
```

Comment:
Why do we even have ofNullable here? What can possibly go wrong?

Reply:
If something goes wrong we should throw exception

Reply:
It's registration Controller I know it's wrong but now I just don't want to "touch' clases not familiar with this branch. I added TODO